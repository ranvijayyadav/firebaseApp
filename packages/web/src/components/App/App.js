//import logo from './logo.svg';
import Login from '../../containers/Login/Login';//components/Login';
import Orders from './../../containers/Orders/Orders';
import ErrorBoundry from '../../containers/ErrorBoundry/ErrorBoundry';
import OrdersDetails from '../../components/OrderDetails/OrderDetails';

//import Error
//import Footer from './../../components/Footer';
import { Switch, Route, BrowserRouter as Router } from 'react-router-dom'
import PrivateRoute from './../../helpers/PrivateRoute';
import { UserProvider } from './../../context/UserContext';

import './App.css';


function App() {
	return (
		// {/* <div className="App"> */}

		<ErrorBoundry>
			<Router>
				<UserProvider>

					<Switch>
						<Route exact path="/login" component={Login} />
						<PrivateRoute path="/order" component={OrdersDetails} />
						<PrivateRoute exact path='/' component={Orders} />
						{/* <PrivateRoute exact path='/orders' component={Orders} /> */}
					</Switch>

				</UserProvider>
			</Router>
		</ErrorBoundry>

		//</div>
	);
}

export default App;
