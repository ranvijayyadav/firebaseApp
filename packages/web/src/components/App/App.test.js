import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

let wrapped = shallow(<App />);
describe('App', () => {
  it("Should render app properly", () => {
    expect(wrapped).toMatchSnapshot();
  })
})
