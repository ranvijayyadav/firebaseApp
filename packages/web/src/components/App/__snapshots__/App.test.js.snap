// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should render app properly 1`] = `
<ErrorBoundry>
  <BrowserRouter>
    <UserProvider>
      <Switch>
        <Route
          component={[Function]}
          exact={true}
          path="/login"
        />
        <PrivateRoute
          component={[Function]}
          path="/order"
        />
        <PrivateRoute
          component={[Function]}
          exact={true}
          path="/"
        />
      </Switch>
    </UserProvider>
  </BrowserRouter>
</ErrorBoundry>
`;
